### PR TITLE
chore(vagrant): install prettier in vagrant

### PR DESCRIPTION
**Pull request by sumo-drosiek**
### Checklist

- [ ] Changelog updated or skip changelog label added
- [ ] Documentation updated
- [ ] Template tests added for new features
- [ ] Integration tests added or modified for major features

### CODE REVIEW

**Review by swiatekm-sumo — APPROVED** [review]
While we're at it, can we do towncrier as well?

**Comment by sumo-drosiek** [comment]
To merge after https://github.com/SumoLogic/sumologic-kubernetes-collection/pull/2921

File: vagrant/provision.sh [hunk 1]
```diff
@@ -3,6 +3,13 @@
 set -euo pipefail
 
 YQ_VERSION=4.30.8
+PRETTIER_VERSION=2.8.3
+HELM_3_VERSION=v3.8.2
+SHELLCHECK_VERSION=v0.7.1
```
[the remark this answers is not in the document]
Comment (sumo-drosiek) [reply vagrant/provision.sh:8]:
```suggestion
SHELLCHECK_VERSION=v0.9.0
```

File: vagrant/provision.sh [hunk 2]
```diff
@@ -3,6 +3,13 @@
...
+K9S_VERSION=v0.24.15
+GO_VERSION=1.18
```
[the remark this answers is not in the document]
Comment (sumo-drosiek) [reply vagrant/provision.sh:10]:
```suggestion
GO_VERSION=1.19
```

**Pull request merged**